build/scripts/Build.fsx

```diff
@@ -35,12 +35,20 @@ let sign file productTitle =
     if release then
         let certificate = getBuildParam "certificate"
         let password = getBuildParam "password"
-        let timestampServer = "http://timestamp.comodoca.com"
+        let timestampServers =
+            [ "http://timestamp.digicert.com" ; "http://timestamp.comodoca.com" ;
+              "http://timestamp.globalsign.com/scripts/timestamp.dll" ; "http://tsa.starfieldtech.com" ;
+              "http://zeitstempel.dfn.de"
+            ]
         let timeout = TimeSpan.FromMinutes 1.
 
-        let sign () =
+        let sign timestampServer =
             let signToolExe = ToolsDir @@ "signtool/signtool.exe"
-            let args = ["sign"; "/f"; certificate; "/p"; password; "/t"; timestampServer; "/d"; productTitle; "/v"; file] |> String.concat " "
+            let args =
+                [ "sign"; "/debug"; "/f"; certificate; "/p"; password
+                  "/t"; timestampServer; "/td"; "SHA256";
```

Comment:
The ODBC PR had `/tr` instead of `/t` here

Reply:
Good catch, just noticed this building an msi locally

```
c:\Projects\elastic\windows-installers\build\tools/signtool/signtool.exe sign /debug /f C:\Users\Mpdre\Downloads\keying\keying\user.p12 /p <redacted> /t http://zeitstempel.dfn.de /td SHA256 /d Elasticsearch /v c:\Projects\elastic\windows-installers\build\out/elasticsearch\7.8.0\elasticsearch-7.8.0.msi
SignTool Error: The /t option is incompatible with the /td option.
```

Updating now